lib/ex_aws/operation/s3.ex

```diff
@@ -40,21 +41,31 @@ defmodule ExAws.Operation.S3 do
       |> operation.parser.()
     end
 
+    def stream!(%{stream_builder: fun}, config) do
```

Comment:
You can do it single line.

```elixir
def stream!(%{stream_builder: fun}, config), do: fun.(config)
```

Reply:
indeed; it was already multi-line, so i left it as-is. may as well tighten it up while i'm in there, though, i supose.